src/Data/Line.purs

```diff
@@ -0,0 +1,93 @@
+{-
+Copyright 2016 SlamData, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-}
+
+module Data.Line (Word, Line, Conf, words, lines, printLine, printWord) where
+
+import Prelude
+
+import Control.Monad.Rec.Class (Step(Loop, Done), tailRec)
```

Comment:
For exports this is totes fine by me. Its also cool if is only one of these unexplicit constructor import. When there is more than one constructor import personally I prefer the explicit constructor imports so it's clear where those constructors are being imported from.